cfme/tests/cloud/test_tenant_quota.py

```diff
@@ -0,0 +1,141 @@
+# -*- coding: utf-8 -*-
+import fauxfactory
+import pytest
+
+from cfme import test_requirements
+from cfme.cloud.instance import Instance
+from cfme.cloud.provider.openstack import OpenStackProvider
+from cfme.services.catalogs.catalog_item import CatalogItem
+from cfme.services.service_catalogs import ServiceCatalogs
+from cfme.utils.generators import random_vm_name
+from widgetastic.utils import Version, VersionPick
```

Comment:
**REQUIRED**
`from widgetastic.utils import Version, VersionPick` should be moved to external import block

Reply:
Moved